cmd/builder/README.md

```diff
@@ -65,6 +65,17 @@ $ ocb --config=builder-config.yaml
 
 Use `ocb --help` to learn about which flags are available.
 
+## Debug
+
+To enable debug for otel-collector-binary set configuration flag `debug-compilation` to true.
+
+Then install `go-delve` and run otel-collector with `dlv` command as following example:
+```bash
+# go install github.com/go-delve/delve/cmd/dlv@latest
+# ~/go/bin/dlv --listen=:2345 --headless=true --api-version=2 --accept-multiclient --log exec .otel-collector-binary -- --config otel-collector-config.yaml
+```
+Finally load otel-collector as a project into IDE, configure debug for Go
```

Comment:
```suggestion
Finally, load the OpenTelemetry Collector as a project in the IDE, configure debug for Go
```

Side note: how to do that? Do you have a link to an external document on how to "load as a project in the IDE"?